src/components/Error.js

```diff
@@ -16,7 +16,7 @@ const Error = props => {
     return (
```

Comment:
where this would render below input or replace input & do we have an rs equivalent for this?